src/Selenium2Library/keywords/_element.py

```diff
@@ -480,7 +480,7 @@ def press_key(self, locator, key):
         """Simulates user pressing key on element identified by `locator`.
 
         `key` is either a single character, or a numerical ASCII code of the key
-        lead by '\\'. In test data, '\\' must be escaped, so use '\\\\'.
+        lead by '\\\\'.
```

Comment:
I have just touched that Press Key documentation and code (it was missing
from an old PR).
With the new Press KeyS keyword we can use named keys instead of escaped
key codes.


On Tue, May 19, 2015 at 2:09 PM, Ed Manlove <notifications@github.com>
wrote:

> In src/Selenium2Library/keywords/_element.py
> <https://github.com/rtomac/robotframework-selenium2library/pull/264#discussion_r30596483>
> :
>
> > @@ -480,7 +480,7 @@ def press_key(self, locator, key):
> >          """Simulates user pressing key on element identified by `locator`.
> >
> >          `key` is either a single character, or a numerical ASCII code of the key
> > -        lead by '\\'. In test data, '\\' must be escaped, so use '\\\\'.
> > +        lead by '\\\\'.
>
> I did have this conflict. Not sure which wording you prefer. My thoughts
> is that if it always has to be escaped there is not point in saying
> otherwise.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtomac/robotframework-selenium2library/pull/264/files#r30596483>
> .
>